### PR TITLE
KOGITO-6130 Review integration test to use Testcontainers

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-runtimes/pull/1743
- https://github.com/kiegroup/kogito-examples/pull/994

### CODE REVIEW

**Review by radtriste — APPROVED** [review]
lgm

File: process-outbox-mongodb-springboot/docker-compose.yml [hunk 1]
```diff
@@ -1,52 +1,69 @@
 version: '2'
 services:
   zookeeper:
-    image: debezium/zookeeper:${DEBEZIUM_VERSION}
+    image: strimzi/kafka:0.20.1-kafka-2.6.0
```
Comment (nmirasch) [inline process-outbox-mongodb-springboot/docker-compose.yml:4]: @cristianonicolai Is this the right image?

File: process-outbox-mongodb-quarkus/docker-compose.yml [hunk 2]
```diff
@@ -1,52 +1,69 @@
 version: '2'
 services:
   zookeeper:
-    image: debezium/zookeeper:${DEBEZIUM_VERSION}
+    image: strimzi/kafka:0.20.1-kafka-2.6.0
```
Comment (nmirasch) [inline process-outbox-mongodb-quarkus/docker-compose.yml:4]: @cristianonicolai Is this the right image?

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: process-outbox-mongodb-quarkus/docker-compose.yml (hunk 2 above)
Comment (cristianonicolai) [reply process-outbox-mongodb-quarkus/docker-compose.yml:4]: @nmirasch yeah, I replaced the image as I was having issues connecting from the test execution, switching to the one we normally use seems to help.

**Review by nmirasch — APPROVED** [review]

**Pull request merged**